tests/lean/interactive/hover.lean.expected.out

```diff
@@ -381,8 +382,7 @@
 {"textDocument": {"uri": "file:///hover.lean"},
  "position": {"line": 200, "character": 2}}
 {"range":
- {"start": {"line": 200, "character": 2},
-  "end": {"line": 200, "character": 15}},
+ {"start": {"line": 200, "character": 2}, "end": {"line": 200, "character": 3}},
```

Comment:
@mhuisi  Here, a narrower range is being highlighted. Before, the whole line got the hover; now, just the leading bullet. This seems consistent with our usual story on tactic highlighting, and OK. Is that the case?

Reply:
I haven't noticed any while working with it interactively, and the test suite didn't find any.

I don't really know how to prove the negative, though. What's your confidence in the test suite's ranges for this kind of thing?

Reply:
> What's your confidence in the test suite's ranges for this kind of thing?

Pretty much zero. Test coverage of the language server is extremely bad. 

> I don't really know how to prove the negative, though.

Well, what caused this smaller range? Is there an informal argument for why the code that causes the smaller range won't negatively affect other examples?

Reply:
CI is giving the smaller range, but I'm getting the original one locally. This is quite concerning. I'll blow away the build directory and try again.

Reply:
> Well, what caused this smaller range? Is there an informal argument for why the code that causes the smaller range won't negatively affect other examples?

OK, worked it out, and the change is gone.

The narrower range was coming because the `TacticInfo` associated with the bullet was being used for a docstring, while on `master`, the docstring is coming from the entire tactic syntax's kind. With the narrower range, the `TacticInfo`'s `ElabInfo` field includes the elaborator used to run the tactic, which is `Lean.Elab.evalTacticCDot`. This inherits the docstring of the syntax `Lean.cdot` due to a change in this PR that I reverted.

I think we're good now.